out of memory: heap=0x%x, size=%d
Second Task_create() failed
Running task1 function
Value is not a multiple of 2
Value returned from canFail1 = -1
Incorrect value used. Must be a multiple of 2
E_notAvailable: Timer not available 0
xdc.runtime.Error.raise: terminating execution
